apps/server/test/lexical/server/code_intelligence/completion/env_test.exs

```diff
@@ -739,4 +864,62 @@ defmodule Lexical.Server.CodeIntelligence.Completion.EnvTest do
                sort_items(items)
     end
   end
+
+  describe "prefix_aliases/1" do
+    test "when in current module" do
+      source = ~q<
+        defmodule User do
+          defstruct [:name, :age]
+
+          def name(%__MODULE__{|}) do
+          end
+        end
+      >
+
+      assert [:__MODULE__] == source |> new_env() |> prefix_aliases()
```

Comment:
ah, I think these should be called `aliases_at_position` or `position_aliases` 
The env's prefix is really just a single line.